packages/paste-core/components/data-grid/__tests__/index.spec.tsx

```diff
@@ -35,22 +36,28 @@ describe('Data Grid', () => {
     });
 
     it('has the correct aria label and role', () => {
-      const {getByTestId} = render(<ComposableCellsDataGrid />);
+      const {getByTestId} = render(<PlainDataGrid />);
       const dataGrid = getByTestId('data-grid');
       expect(dataGrid.getAttribute('aria-label')).toBeDefined();
       expect(dataGrid.getAttribute('role')).toBe('grid');
     });
   });
 
-  describe('Keyboard navigation', () => {
-    it('moves with arrow keys in navigational mode', () => {
+  describe('Composable Cells functionality', () => {
+    it('has proper keyboard navigation behavior', () => {
       const {getByTestId} = render(<ComposableCellsDataGrid />);
+      const wrapper = getByTestId('data-grid');
       const headerCell = getByTestId('header-1');
-      const firstInputCell = getByTestId('input-0-0')?.parentElement?.parentElement;
+      const firstRowFirstInputCell = getByTestId('input-0-0');
+      const firstRowSecondInputCell = getByTestId('input-0-1');
+      const secondRowFirstInputCell = getByTestId('input-1-0');
+      const firstInputCell = firstRowFirstInputCell?.parentElement?.parentElement;
+
       if (firstInputCell == null) {
```

Comment:
Not blocking, just info: I think this line could be removed if on lines 51, 52, 53 we modified them to look liike:

```
it('has proper keyboard navigation behavior', async () => {
    const firstRowFirstInputCell = await screen.findByTestId('input-0-0');
});
 ```

[More context in this article.](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find)